sdk/src/main/java/com/sygic/travel/sdk/di/networkModule.kt

```diff
@@ -0,0 +1,29 @@
+package com.sygic.travel.sdk.di
+
+import android.content.Context
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.github.salomonbrys.kodein.singleton
+import com.google.gson.FieldNamingPolicy
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import okhttp3.Cache
+import okhttp3.logging.HttpLoggingInterceptor
+
+internal val networkModule = Kodein.Module {
+	bind<Cache>() with singleton {
+		val cacheFile = instance<Context>().cacheDir
+		Cache(cacheFile, 10 * 1024 * 1024L) // 10 MB Cache
+	}
+
+	bind<Gson>() with singleton {
+		GsonBuilder()
+			.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+			.create()
+	}
+
+	bind<HttpLoggingInterceptor>() with singleton {
```

Comment:
I would put this into st api module

Reply:
I doesn't put it there because `LoggingInterceptor` will be used also in SsoModule which will follow.